src/Engine/RadiumEngine.hpp

```diff
@@ -198,6 +275,16 @@ class RA_ENGINE_API RadiumEngine
 
     /// For internal resources management in a filesystem
     std::string m_resourcesRootDir;
+
+    Scalar m_dt;                ///< The time delta between 2 consecutive frames.
+    Scalar m_startTime{0_ra};   ///< The `start` time for the time window.
+    Scalar m_endTime;           ///< The `end` time for the time window.
+    Scalar m_time{0_ra};        ///< The current time.
+    bool m_realTime{false};     ///< Whether we use the effective time flow or the constant one.
+    bool m_play{false};         ///< Shall time flow.
+    bool m_step{true};          ///< Shall time flow for only one frame.
+    bool m_pingPongMode{false}; ///< Is PingPong mode enabled.
```

Comment:
All of the above time related members might be packed together into an internal struct TimeMode or something similar ?